File: SofiForce.Models/Models/EntityModels/ClientGroupModel.cs
using System;

namespace Models
{
    public class ClientGroupModel
    {
        public int ClientGroupId { get; set; }
        public string ClientGroupCode { get; set; }
        public string ClientGroupNameEn { get; set; }
        public string ClientGroupNameAr { get; set; }
        public bool? IsActive { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public string Icon { get; set; }
        public string Color { get; set; }
        public int? DisplayOrder { get; set; }

        public string Notes { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/ClientGroupModel.ts
import { ClientModel } from "./clientModel";
import { ClientGroupSubModel } from "./clientGroupSubModel";

export interface ClientGroupModel {
    clientGroupId: number;
    clientGroupCode: string;
    clientGroupNameEn: string;
    clientGroupNameAr: string;
    isActive: boolean | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    icon: string;
    color: string;
    displayOrder: number | null;
    notes:string;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
}